app/controllers/application_controller.rb

```diff
@@ -1,10 +1,15 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :create_cart
```

Comment:
Yes why wouldn't we do this when he adds the first item to the cart? That's how I set up this feature.